src/resources/accounts.js

```diff
@@ -64,6 +64,53 @@ class Accounts {
     }
   }
 
+  async createDepositAddress(account = {}, ticker) {
+    if (!account.id) {
+      throw new SDKError(400, '[Zabo] Missing `id` property in `account` object. See: https://zabo.com/docs#create-a-deposit-address')
+    } else if (!account.token) {
+      throw new SDKError(400, '[Zabo] Missing `token` property in `account` object. See: https://zabo.com/docs#create-a-deposit-address')
+    } else if (!ticker || typeof ticker !== 'string') {
+      throw new SDKError(400, '[Zabo] Missing or invalid `ticker` parameter. See: https://zabo.com/docs#create-a-deposit-address')
+    }
+
+    const providersWithStaticDepositAddresses = [
```

Comment:
Ref this, I wonder how this can be scaled. Not sure we can list providers individually as the list grows...